Makefile

```
@@ -309,9 +319,10 @@ flexlink: flexdll/Makefile
 flexlink.opt:
 	cd flexdll && \
 	mv flexlink.exe flexlink && \
-	($(MAKE) OCAML_FLEXLINK="../boot/ocamlrun ./flexlink" MSVC_DETECT=0 \
+	($(MAKE) OCAML_FLEXLINK="../boot/ocamlrun$(EXE) \
+	  ./flexlink" MSVC_DETECT=0 \
```

You have cut the line in a very strange place.
```suggestion
	($(MAKE) OCAML_FLEXLINK="../boot/ocamlrun$(EXE) ./flexlink" \
	           MSVC_DETECT=0 \
```